src/components/Quiz/QuizManager/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { useMemo, useState } from "react"
+import { FaGithub } from "react-icons/fa"
+import { Box, Flex, Icon, Stack, Text, useDisclosure } from "@chakra-ui/react"
+
+import { QuizStatus, QuizzesSection } from "@/lib/types"
+import { RawQuizzes } from "@/lib/interfaces"
+
+import { ButtonLink } from "@/components/Buttons"
+import Translation from "@/components/Translation"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+
+import { INITIAL_QUIZ } from "@/lib/constants"
+
+import QuizWidget from "../QuizWidget"
+import QuizzesList from "../QuizzesList"
+import QuizzesModal from "../QuizzesModal"
+import QuizzesStats from "../QuizzesStats"
+import { useLocalQuizData } from "../useLocalQuizData"
+
+const handleGHAdd = () =>
+  trackCustomEvent({
+    eventCategory: "quiz_hub_events",
+    eventAction: "Secondary button clicks",
+    eventName: "GH_add",
+  })
+
+type QuizPageKeys = "quizzes-stats"
```

Comment:
Proposal: Have a single source of truth (as a string union type) for storage key names for each page the uses `QuizManager` as a way to help prevent unexpected changes that result in typos of the keys. (Avoid negative impact on users with existing stats)

Definitely a matter of preference here if this is going to help.

Also need to consider a way to replace `quizzes-stats` with a key name more unique to the hub page so it stands out from key names supplied for other pages using `QuizManager`.